libfdo-data/Cargo.toml

```diff
@@ -10,7 +10,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 fdo-data-formats = { path = "../data-formats", version = "0.1.0" }
-
+fdo-util = { path = "../util", version = "0.1.0" }
```

Comment:
Why are you adding this dependency to `libfdo-data`?

Reply:
That's a very good question I don't have an answer to :laughing:. Blue screen of death moment in my mind, I guess. Removed.